screen-accounts/src/main/java/com/ivy/accounts/AccountsTab.kt

```diff
@@ -279,16 +279,16 @@ private fun AccountHeader(
             }
         }
 
-        Spacer(Modifier.height(4.dp))
+        Spacer(Modifier.height(2.dp))
 
         BalanceRow(
             modifier = Modifier
                 .align(Alignment.CenterHorizontally)
                 .clickableNoIndication {
                     onBalanceClick()
                 },
-            decimalPaddingTop = 7.dp,
-            spacerDecimal = 6.dp,
+            decimalPaddingTop = 1.dp,
```

Comment:
Increase the decimalPaddingTop because the 0.00 looks weird

Reply:
**Increased the decimalPaddingTop to 7.dp**
__________________________________________________

![WhatsApp Image 2023-10-14 at 14 56 13](https://github.com/Ivy-Apps/ivy-wallet/assets/71680976/e9ee1f0f-f973-4f83-a2a7-7e6ca56b3869)